packages/babel-preset/index.js

```diff
@@ -73,15 +73,10 @@ module.exports = function shopifyCommonPreset(
     // Enable loose mode to use assignment instead of defineProperty when typescript is enabled
```

Comment:
These comments need to be updated as we're not controling this by using loose options anymore